README.md

```diff
@@ -47,15 +61,21 @@
 ### Key Features <a name="key-features"></a>
 
 - **A single page app built with React**
-- **Ability to route between multiple pges**
+- **Using Redux for state management**
+- **Using React router for routing**
+- **Using Redux Middleware**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- LIVE DEMO -->
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
```

Comment:
- [ ] Kindly consider your project should be deployed and accessible online using [Render](https://render.com/), [Netlify](https://www.netlify.com/) or [GitHub Pages](https://pages.github.com/) and add a link to the online version of your app to your README file.